stablehlo/dialect/VhloAttrs.td

```diff
@@ -203,4 +166,55 @@ def VHLO_ConvolutionAttributes {
   );
 }
 
+//===----------------------------------------------------------------------===//
+// Forked Attributes
+//===----------------------------------------------------------------------===//
+
+def VHLO_ArrayAttr : VHLO_AttrDef<"ArrayV1"> {
```

Comment:
This is more important for types/attrs than ops because it gets very confusing what `ArrayAttr` is referring to in a namespace, since we nest namespaces `namespace mlir { namespace vhlo { ... ArrayAttr::get(...)  }}` in the same namespace. If we include the file with `vhlo::ArrayAttr` this is resolved to VHLO, otherwise it gets resolved to `mlir::ArrayAttr`. Adding a V1 to the type name prevents any ambiguity. This impacts ops less since we don't nest namespace VHLO/StableHLO anywhere.

I'll update all names to be `VHLO_ArrayAttrV1`, but will consider suffixing all ops in a later submission.